tests/system/testdata/events/list.txtar

```diff
@@ -1,38 +1,38 @@
-# Negative tests: empty events list, with/out --fail flag.
-ak event list --integration http
+ak deploy --manifest project.yaml
```

Comment:
Please back the negative tests: empty events list, with/out --fail flag (using the test integration maybe?)

Reply:
done